.github/workflows/main.yml

```diff
@@ -1,41 +1,64 @@
 name: CI
 
-on: [push, pull_request]
+on: [ push, pull_request ]
+
+permissions:
+  contents: read
 
 jobs:
   lint-pr:
     if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: TrueBrain/actions-flake8@v1.4.1
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
```

Comment:
Without setting this, every action below has implict access to $GITHUB_TOKEN. That's not necessary.

See https://github.com/actions/checkout/issues/485